R/ComponentScores.R

```diff
@@ -39,6 +39,10 @@ ScoreUsingSavedComponent <- function(seuratObj, componentOrName, fieldName) {
   seuratObj <- Seurat::AddMetaData(seuratObj, cellScores, col.name = fieldName)
 
   print(FeaturePlot(seuratObj, features=fieldName, order = T, cols = c("navy", "dodgerblue", "gold", "red")))
+  print(FeaturePlot(seuratObj, features = fieldName, order = T))
```

Comment:
@eisascience, i have no problem adding a histogram; however:

- please remove the duplicate FeaturePlot
- SerObj isnt a valid variable